{{ cookiecutter.folder_name }}/README.rst

```diff
@@ -2,12 +2,19 @@
 {{ cookiecutter.project_name }}
 ===============================
 
+.. image:: https://img.shields.io/travis/{{ cookiecutter.github_repo_group }}/{{ cookiecutter.repo_name }}.svg
+        :target: https://travis-ci.org/{{ cookiecutter.github_repo_group }}/{{ cookiecutter.repo_name }}
+
+.. image:: https://img.shields.io/pypi/v/{{ cookiecutter.repo_name }}.svg
```

Comment:
We rarely deploy on `PyPI`

Reply:
I can remove it. However, my opinion is that we should be, and this could be an impetus to do so ("_go away red README.rst marker!_")